extensions/amp-analytics/0.1/vendors/kenshoo.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const KENSHOO_CONFIG = /** @type {!JsonObject} */ ({
+  'vars': {
+    'clientId': 'CLIENT_ID(ken_amp_id)',
+    'channelClickId': 'QUERY_PARAM(gclid)',
+  },
+  'requests': {
+    'host': 'https://amp.xg4ken.com',
+    'parameters':
+      'ampcid=${clientId}' +
+      '&chcid=${channelClickId}' +
+      '&tid=${tid}' +
```

Comment:
So it seems like the `tid` is the reason why the tests are failing.

Is this a user provided value?

If it is a user provided value, simply fix the `extensions/amp-analytics/0.1/test/vendor-requests.json` to match the value that is expected when the travis runs.

Thank you! 😄

Reply:
Thanks!
It's indeed the problem. (I changed these parameters behavior few times and forgot to update the tests..)